gateleen-hook/src/main/resources/gateleen_hooking_schema_hook

```diff
@@ -131,6 +135,32 @@
                 "header",
                 "value"
             ]
-        }
+        },
+		"ProxyOptions": {
+			"properties": {
+				"type": {
+                    "type": "string",
+                    "enum": [
+                        "HTTP",
+                        "SOCKS4",
+                        "SOCKS5"
+                    ]
+				},
+				"host": {
+					"type": "string"
```

Comment:
same comments as for the routing-rule-schema.
It would be good to specify "required" attributes